packages/modules/src/dynamicModules.ts

```diff
@@ -85,15 +85,22 @@ export interface IConfigurableDynamicRootModule<T, U> {
 
   forRootAsync(
     moduleCtor: Type<T>,
-    asyncModuleConfig: AsyncModuleConfig<U>
+    asyncModuleConfig: AsyncModuleConfig<U>,
   ): DynamicModule;
 
   externallyConfigured(
     moduleCtor: Type<T>,
-    wait: number
+    wait: number,
   ): Promise<DynamicModule>;
 }
 
+/**
+ *
+ * @deprecated Please use Nestjs Configurable Module {@see https://docs.nestjs.com/fundamentals/dynamic-modules#configurable-module-builder}
```

Comment:
I have added a deprecation/redirect to the nestjs module builder. We could potentially remove this module as it's not worth to be maintaining since Nestjs now has a better alternative